### PR TITLE
SUBMARINE-1157. Serving model via Python API or CLI

### PR DESCRIPTION
**Pull request by KUAN-HSUN-LI**
### What is this PR for?
Create the Python API and CLI command for serving a registered model.

### What type of PR is it?
[Feature]

### Todos
* [x] - Save Pytorch model via torchscript
* [x] - Generate the Serve API
* [x] - Transfer the model description file to the config which is used in the Triton Inference Server. (It will be handled in this issue https://issues.apache.org/jira/browse/SUBMARINE-1173)
### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-1157
### How should this be tested?
The tests will provide in another PR because the transfer of the config is not finished.
### Screenshots (if appropriate)
https://user-images.githubusercontent.com/38066413/147928471-7e8cc70b-99df-4b49-89bb-bc1627bbb885.mp4

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? Yes

### CODE REVIEW

**Comment by KUAN-HSUN-LI** [comment]
@jeff-901 @atosystem Please help me review the PR. Thanks!

File: submarine-sdk/pysubmarine/submarine/cli/main.py [hunk 1]
```diff
@@ -67,6 +73,11 @@ def cmdgrp_config():
 cmdgrp_list.add_command(environment_cmd.list_environment)
 cmdgrp_get.add_command(environment_cmd.get_environment)
 cmdgrp_delete.add_command(environment_cmd.delete_environment)
+# # serve
```
Comment (atosystem) [inline submarine-sdk/pysubmarine/submarine/cli/main.py:76]: [Trivial] Delete one '#'?

File: submarine-sdk/pysubmarine/submarine/cli/serve/command.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    console.print("[bold red] Timeout!")
+                    return
+
+        result = thread.get()
+        click.echo(result)
+
+        if wait:
+            if result["status"] == "Deleted":
+                console.print(
+                    f"[bold green] Serve: model name:{model_name}, version: {model_version}"
```
Comment (jeff-901) [inline submarine-sdk/pysubmarine/submarine/cli/serve/command.py:109]: Maybe we can console "f"[bold green] Serve Deleted: model name:{model_name}, version: {model_version}""

**Review by jeff-901 — COMMENTED** [review]
Other LGTM.

**Review by pingsutw — COMMENTED** [review]
Thank you, LGTM. 
Could you resolve the conflict?

**Review by pingsutw — APPROVED** [review]

**Pull request closed without merging**